service-tests/suggest.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'suggest', title: 'suggest', pathPrefix: '/$suggest' });
+module.exports = t;
+
+
+t.create('issues')
+  .get('/v1?url=' + encodeURIComponent('https://github.com/atom/atom'))
+  .addHeader('origin', 'https://shields.io')
```

Comment:
https://github.com/badges/shields/blob/master/lib/suggest.js#L15 requires this header value.

Reply:
Worth mentioning that in a comment? ^^